src/wp-admin/css/customize-nav-menus.css

```diff
@@ -665,13 +665,17 @@
 #available-menu-items .menu-item-handle.item-added .item-title,
 #available-menu-items .menu-item-handle.item-added:hover .item-add,
 #available-menu-items .menu-item-handle.item-added .item-add:focus {
-	color: #8c8f94;
+	color: #1d2327;
```

Comment:
```suggestion
	color: #646970;
```

`#646970`, or [Gray 50](https://codepen.io/ryelle/full/WNGVEjw), would have sufficient contrast of 5.53:1 against white. It also matches the shade used for the item type (Page, Custom Link, etc.).

<img width="599" height="298" alt="Added menu item with Gray 50" src="https://github.com/user-attachments/assets/df181df8-a759-410d-8ab3-3d63ab7eb0dc" />

I prefer to follow the original intent of making the added items a little lighter. If changing all the text to gray does not improve the interface, however, the entire ruleset could be removed instead of matching the darker gray. Then the menu item would still have the lighter text for the item type and blue text when hovering over the menu item.